setup.py

```diff
@@ -48,7 +48,7 @@
                         'protobuf<5.0.0',
                         'requests>=2.6.0',
                         'six>=1.12.0',
-                        'urllib3<1.27,>=1.26.5',],
+                        'urllib3>=1.26.18',],
```

Comment:
The way I see there is no reason why customers who had to stick with `urllib3`  version `1.26.5`, should not be able to use our tracer without any issue. So we perhaps we could keep this on `>=1.26.5`?

Reply:
Makes sense 👍🏼